src/pages/resultsView/ResultsViewURLWrapper.ts

```diff
@@ -389,18 +401,37 @@ export default class ResultsViewURLWrapper
         );
     }
 
-    @computed public get comparisonSubTabId() {
-        return this.query.comparison_subtab || GroupComparisonTab.OVERLAP;
-    }
-
     @autobind
     public setTabId(tabId: ResultsViewTab, replace?: boolean) {
         this.updateURL({}, `results/${tabId}`, false, replace);
     }
 
+    public get getSubTab() {
+        const tabSegment = this.getCurrentTabSegment();
+        if (!tabSegment) return '';
```

Comment:
minor thing but I think we should avoid one-liner if statements 

```suggestion
        if (!tabSegment) {
            return '';
        }
```

Reply:
Done